src/azure-cli/setup.py

```diff
@@ -56,6 +56,7 @@
     'azure-cli-core=={}.*'.format(VERSION),
     'azure-cli-nspkg~=3.0,>=3.0.3',
     'azure-cli-telemetry>=1.0.2,<2.0',
+    'azure-core>=1.2.2',
```

Comment:
remove this one once your package is released to pypi. use released package instead, and you also need add that released package in requirements.xxx.txt.